src/components/cart/CartItem.ts

```diff
@@ -13,6 +13,7 @@ class CartItem extends Base<HTMLDivElement> {
   private ticket: CartTicket;
   private seatPriceTd: HTMLTableCellElement;
   private currency: string;
+  private currencyBehind: boolean;
```

Comment:
A better alternative and naming for this property could be:

`private currencyPosition: 'start' | 'end';`